src/extension/background-script/actions/allowances/__tests__/add.test.ts

```diff
@@ -1,25 +1,13 @@
 import db from "~/extension/background-script/db";
+import { getallowanceFixtureWithfilter } from "~/fixtures/allowances";
 import type { DbAllowance, MessageAllowanceAdd } from "~/types";
 
 import addAllowance from "../add";
 
 Date.now = jest.fn(() => 1487076708000);
 
-const mockAllowances: DbAllowance[] = [
-  {
-    enabled: true,
-    host: "pro.kollider.xyz",
-    id: 3,
-    imageURL: "https://pro.kollider.xyz/favicon.ico",
-    lastPaymentAt: 0,
-    lnurlAuth: true,
-    name: "pro kollider",
-    remainingBudget: 500,
-    totalBudget: 500,
-    createdAt: "123456",
-    tag: "",
-  },
-];
+const mockAllowances: DbAllowance[] =
```

Comment:
Instead of using this rather confusing filter could we just use something like this?:
```ts
const mockAllowances: DbAllowance[] = [{...allowanceFixture[0]}]
```